README.md

```diff
@@ -1,21 +1,58 @@
 # DEVit
-DEVit's website powered by Gatsby and other amazing open source projects.
 
-## Install
+DEVit's website is powered by Gatsby and other amazing open source projects.
 
-Make sure that you have install dependencies.
+## How to contribute
+
+### Install
+
+The site is made with [Gatsby](https://www.gatsbyjs.org).
+
+You need `node`, `gatsby` and `yarn` installed in order to build the site.
+
+To check if you have the necessary programs mentioned above installed and to see their version you can run on a terminal:
+`<PROGRAM> --version`, e.g. *node --version*
+
+* Install `node`
+To install node you can visit [https://nodejs.org/en/download/](https://nodejs.org/en/download/).
```

Comment:
So should I leave it as is or what do you suggest doing?

Reply:
Honestly, it might be a bit too complex for the people that just joined so Node.js is fine (but mention the version they need to install) and then create a new section called "Advanced topics" and suggest nvm there.